go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/mailgun/holster/v4
 
-go 1.21
-
-toolchain go1.21.12
+go 1.22.0
```

Comment:
Could you update the https://github.com/mailgun/holster/blob/master/.github/workflows/test.yaml#L19 too?

Reply:
Done.  I set it for 1.22, 1.23, 1.24.